client/src/components/ConsolePage/GeoView.scss

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Dgraph Labs, Inc. and Contributors
+//
+// Licensed under the Dgraph Community License (the "License"); you
+// may not use this file except in compliance with the License. You
+// may obtain a copy of the License at
+//
+//     https://github.com/dgraph-io/ratel/blob/master/LICENSE
+
+@import url("https://unpkg.com/leaflet@1.6.0/dist/leaflet.css");
```

Comment:
Please import it from the node_modules, not from unpkg, I've pasted an example of doing it above.
Import from CDN won't work offline, and we have (semi?)-official support of ratel on admin machines w/o internet access.